Tests/LocheckLogicTests/LocalizedStringPairTests.swift

```diff
@@ -68,11 +71,9 @@ class LocalizedStringPairTests: XCTestCase {
                 FormatArgument(specifier: "@", position: 1, isPositionExplicit: false),
                 FormatArgument(specifier: "@", position: 2, isPositionExplicit: true),
             ])
-        XCTAssertTrue(problemReporter.problems.isEmpty)
```

Comment:
I might just be missing something, but why were these asserts removed?

Reply:
They weren't doing anything. The ProblemReporter was never used.